components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -175,10 +183,19 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                 >
                     {hasSubmenu ? subMenu!.map((s) => {
                         const hasDivider = s.id === 'ChannelMenu-moveToDivider';
+                        let aria = ariaLabel;
+                        if (s.action) {
+                            aria = s.text === selectedValueText ?
+                                s.text + ' ' + Utils.localizeMessage('sidebar.menu.item.selected', 'selected') :
+                                s.text + ' ' + Utils.localizeMessage('sidebar.menu.item.notSelected', 'not selected');
+                        }
                         return (
                             <span
                                 className={classNames(['SubMenuItemContainer', {hasDivider}])}
                                 key={s.id}
+                                onFocus={this.setBackGroundColor}
+                                onBlur={this.unsetBackGroundColor}
+                                tabIndex={s.tabIndex}
```

Comment:
This doesn't seem to be working since I can still tab to the divider. Even if we set this to -1, there's a div rendered by the `SubMenuItem` that has a `tabIndex` set, so I'm still able to focus it

![Screen Shot 2022-10-03 at 3 09 40 PM](https://user-images.githubusercontent.com/3277310/193663555-69d22c82-e6d9-47f2-a609-4df1d37f45e6.png)

Maybe the SubMenuItem itself needs to be the one that figures out if it should have a `tabIndex` or not?